### PR TITLE
New UI topic #68

### LINKED ISSUE #68: create_topics_new_ui
- Create Navbar
- Add Better frontend for login
- Show Topics with proficiency graphics

### PR DESCRIPTION
**Pull request by aravindsakthivel**
- Added Navbar
- Added new UI

### CODE REVIEW

File: client/web/src/Structure/Results Display/State/action.js [hunk 1]
```diff
@@ -2,7 +2,6 @@ import { resultConstant } from "./actionTypes";
 import axios from "axios";
 
 const RESULT_API = process.env.REACT_APP_ATTEMPT_URL;
-const token = localStorage.getItem("token");
```
Comment (arjun1237) [inline client/web/src/Structure/Results Display/State/action.js:5]: use enums for token names... i have already created enum folder. Use that.
Also the localstorage methods are in utils, use that

File: client/web/src/Structure/Results Display/State/action.js [hunk 2]
```diff
@@ -20,6 +19,7 @@ export const getResultFailure = (payload) => ({
 
 export const getResult = (payload) => (dispatch) => {
   dispatch(getResultRequest())
+  const token = localStorage.getItem("token");
```
Comment (arjun1237) [inline client/web/src/Structure/Results Display/State/action.js]: use enums for token names... i have already created enum folder. Use that.
Also the localstorage methods are in utils, use that

File: client/web/src/Structure/Topics/State/action.js [hunk 3]
```diff
@@ -22,7 +21,7 @@ const getTopicsFailure = (payload) => ({
 
 const getTopics = () => (dispatch) => {
   dispatch(getTopicsLoading());
-
+  const token = localStorage.getItem("token");
```
Comment (arjun1237) [inline client/web/src/Structure/Topics/State/action.js]: here again you need to use enums and already provided methods

File: client/web/src/Structure/Topics/Components/startQuizModal.jsx [hunk 4]
```diff
@@ -21,9 +22,9 @@ const StartQuizModal = ({ open, handleClose, topic, topicId }) => {
   const isSuccessQuiz = useSelector((state) => state.topics.isSuccessQuiz);
   const startQuiz = () => {
     dispatch(topicActions.attemptQuiz(topicId)).then((res) => {
-      if (res === "success") {
+      if (res.final === "success") {
         history.push("/quiz_questions");
-      } else {
+      } else if (res.final === "failure") {
         alert("Unable to start Quiz try later");
```
Comment (arjun1237) [inline client/web/src/Structure/Topics/Components/startQuizModal.jsx:28]: why dont you just return res.final as Boolean

Comment (aravindsakthivel) [reply client/web/src/Structure/Topics/Components/startQuizModal.jsx:28]: I always use success and failure for easy understanding

Comment (arjun1237) [reply client/web/src/Structure/Topics/Components/startQuizModal.jsx:28]: ok

**Pull request merged**